File: back/src/ProAtividade.API/Models/Atividade.cs
namespace ProAtividade.API.Models
{
    public class Atividade
    {
        public long Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public PrioridadeEnum Prioridade { get; set; }

        public Atividade()
        {
            
        }

        public Atividade(long id)
        {
            this.Id = id;
        }
    }
}

File: front/src/models/atividade.ts
export interface IAtividade {
    id: number,
    prioridade: PrioridadeEnum,
    titulo: string,
    descricao: string
}

export enum PrioridadeEnum {
    NaoDefinido = 'NaoDefinido',
    Baixa = 'Baixa',
    Normal = 'Normal',
    Alta = 'Alta'
}